src/db/query.js

```diff
@@ -1,4 +1,5 @@
-import { fireEvent } from '../event'
+import {fireEvent} from '../event'
+import {PLATFORM} from "@/platform";
```

Comment:
Also this alias will collide with the consumer project alias, this should be relative path.